sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -224,12 +224,18 @@ def _build_ssl_opts(self, sslopts):
                 "SSL configuration must be a dictionary, or the value True."
             ) from None
 
-    def _build_ssl_context(
+    async def load_verify_locations(self, ctx, cafile=None):
+        if not cafile:
+            cafile = certifi.where()
+        loop = asyncio.get_event_loop()
+        await loop.run_in_executor(None, ctx.load_verify_locations, cafile)
```

Comment:
Just use asyncio.to_thread

Reply:
@graingert - If we allow users to pass in their own SSLContext, would asyncio.to_thread be needed?

Reply:
Users would need to create their own SSLContext in their own asyncio.to_thread